DigitalOffice/Tests/UserServiceTests/Validators/UserCreateRequestValidatorTests.cs

```diff
@@ -3,6 +3,7 @@
 using NUnit.Framework;
 using LT.DigitalOffice.UserService.RestRequests;
 using LT.DigitalOffice.UserService.Validators;
+using System;
```

Comment:
Using directive is not required by the code and can be safely removed

Reply:
fixed